extensions/data-transfer/portability-data-transfer-koofr/src/main/java/org/datatransferproject/transfer/koofr/videos/KoofrVideosImporter.java

```diff
@@ -77,19 +81,20 @@ public ImportResult importItem(
           album.getId(), album.getName(), () -> createAlbumFolder(album, koofrClient));
     }
 
+    final LongAdder totalImportedFilesSizes = new LongAdder();
     for (VideoModel videoModel : resource.getVideos()) {
-      String id;
-      if (videoModel.getAlbumId() == null) {
```

Comment:
Had to remove it because importAndSwallowIOExceptions only accepts ImportableItem type. VideoItem type already seems to be implementing that type and since the only point of idempotent id is being unique, I thought it should be fine without needing any changes.